Mathlib/CategoryTheory/Monoidal/Cartesian/Grp_.lean

```diff
@@ -121,12 +127,100 @@ lemma essImage_yonedaGrp :
     letI := Grp_Class.ofRepresentableBy X F e
     exact ⟨.mk' X, ⟨yonedaGrpObjIsoOfRepresentableBy X F e⟩⟩
 
+@[reassoc]
+lemma Grp_Class.inv_comp (f : X ⟶ G) (g : G ⟶ H) [IsMon_Hom g] : f⁻¹ ≫ g = (f ≫ g)⁻¹ := by
+  simp [Hom.inv_def, IsMon_Hom.inv_hom]
+
+@[reassoc]
+lemma Grp_Class.div_comp (f g : X ⟶ G) (h : G ⟶ H) [IsMon_Hom h] :
+    (f / g) ≫ h = (f ≫ h) / (g ≫ h) :=
+  ((yonedaGrp.map (Grp_.homMk h)).app (.op X)).hom.map_div f g
+
+@[reassoc]
+lemma Grp_Class.zpow_comp (f : X ⟶ G) (n : ℤ) (g : G ⟶ H) [IsMon_Hom g] :
+    (f ^ n) ≫ g = (f ≫ g) ^ n :=
+  ((yonedaGrp.map (Grp_.homMk g)).app (.op X)).hom.map_zpow f n
+
```

Comment:
Maybe it should be documented somewhere that these lemmas (and some of the ones below) are aligned with `MonoidHom.map_inv`, `Monoiidhom.map_div` etc. so that `_ ≫ g` has the same API as a `MonoidHom`, to justify the choice of what is in the LHS/RHS, and what is/isn’t simp lemmas.

Reply:
Why not putting it next to the concerned code? I'm not asking for a long library note or doc entry, just something like `-- the next lemmas are aligned with MonoidHom.`.